### ISSUE TITLE
Stop editing list property or string property does not reset input to initial value

### ISSUE THREAD

**Issue by WeirdRubberDuck** [OpenSpace/OpenSpace#2496]
It should. Just like the numeric properties do when entering a value using text. 

The result is very confusing. As the input has another value than the property actually does

**Comment by WeirdRubberDuck** [comment]
Actually, the NumericInput properties sets the value on blur, as long as it's a value in the min and max range. It should only set the value on enter, not on blur

Blurring should always reset the previous property value

**Comment by alexanderbock** [comment]
We should discuss how we want the properties to behave.  When a user clicks away from a property field after changing a value, what should happen?

**Comment by engbergandreas** [comment]
Fixed in rewrite https://github.com/OpenSpace/OpenSpace/commit/e3f5b83f7476c8507dccd45ab98e4731504af532?

**Comment by alexanderbock** [comment]
Would agree

**Comment by WeirdRubberDuck** [comment]
Yes - the behavior on blur is more consistent in the rewritten UI
